crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -1087,22 +1083,20 @@ impl HolochainP2pHandler for HolochainP2pActor {
         let payload = crate::wire::WireMessage::get_links(link_key, r_options).encode()?;
 
         let kitsune_p2p = self.kitsune_p2p.clone();
+        let tuning_params = self.tuning_params.clone();
         Ok(async move {
+            let mut input = kitsune_p2p::actor::RpcMulti::new(
+                &tuning_params,
+                space,
+                from_agent,
+                basis,
+                payload,
+            );
             // TODO - We're just targeting a single remote node for now
             //        without doing any pagination / etc...
             //        Setting up RpcMulti to act like RpcSingle
-            let result = kitsune_p2p
-                .rpc_multi(kitsune_p2p::actor::RpcMulti {
-                    space,
-                    from_agent,
-                    basis,
-                    remote_agent_count: Some(1),
-                    timeout_ms: options.timeout_ms,
-                    as_race: false,
-                    race_timeout_ms: options.timeout_ms,
-                    payload,
-                })
-                .await?;
+            input.max_remote_agent_count = 1;
```

Comment:
What's the reason for not attempting more then one?

Reply:
I actually have no idea... We set the remote count to 1 in the old code, so I was just trying not to change anything.